ani-cli

```diff
@@ -134,20 +134,24 @@ provider_init() {
     provider_id=$(printf "%s" "$resp" | sed -n "$2" | head -1 | cut -d':' -f2)
 }
 
-hex_to_ascii() {
-    hex="$1"
-    len=$(printf "%s" "$hex" | wc -c)
-    ascii=""
-    i=1
-    while [ "$i" -lt "$len" ]; do
-        char=$(printf "%s" "$hex" | cut -c "$i-$((i + 1))")
-        dec=$(printf "%d" "0x$char")
-        oct=$(printf "%03o" "$dec")
-        # shellcheck disable=SC2059
-        ascii="$ascii$(printf "\\$oct")"
-        i=$((i + 2))
+bypass_debug() {
+    if printf "%s" "$-" | grep -q 'x'; then
+        set +x
+    elif [ "$1" -eq "0" ]; then
+        set -x
+    fi
+}
```

Comment:
This is no longer necessarily a function as we only `set +x` in line 146 and `set -x` in 154. Minus 8 lines, and if we want to do it again we can still re-introduce the function.

Reply:
yeah, I wrote that function cause shellcheck was complaining of && || , and elif is necessary cause If it's normal else then it would output debug value in non-debug mode
![image](https://github.com/pystardust/ani-cli/assets/76195824/fd0dde04-074e-4799-a277-7857d0a2d074)

Reply:
removed the function and replaced it with one-liners